src/components/TabSelector/TabIcon.js

```diff
@@ -0,0 +1,49 @@
+import {StyleSheet, View, Animated} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+import themeColors from '../../styles/themes/default';
+
+const propTypes = {
+    /** Icon to display on tab */
+    icon: PropTypes.func,
+
+    /** Animated opacity value while the label is inactive state */
+    // eslint-disable-next-line
+    inactiveOpacity: PropTypes.any,
```

Comment:
Why `PropTypes.any`? couldn't be `PropTypes.number`? As I see, these props are numbers 0 or 1?